packages/astro/astro-jsx.d.ts

```diff
@@ -533,18 +534,23 @@ declare namespace astroHTML.JSX {
 			| undefined
 			| null;
 		is?: string | undefined | null;
+
+		// Microdata API
 		itemid?: string | undefined | null;
 		itemprop?: string | undefined | null;
 		itemref?: string | undefined | null;
 		itemscope?: boolean | string | undefined | null;
 		itemtype?: string | undefined | null;
+
 		lang?: string | undefined | null;
+		part?: string | undefined | null;
+		popover?: boolean | string | undefined | null;
 		slot?: string | undefined | null;
 		spellcheck?: 'true' | 'false' | boolean | undefined | null;
 		style?: string | StyleObject | undefined | null;
 		tabindex?: number | string | undefined | null;
 		title?: string | undefined | null;
-		translate?: 'yes' | 'no' | undefined | null;
+		translate?: 'yes' | 'no' | string | undefined | null;
```

Comment:
Unrelated to the additions, but `translate` can also be an empty string (equivalent in behaviour to `yes`, weird)

Reply:
Would `string` be the correct type here, in that case? Seems like it should only accept `''`?

Reply:
Ah, this is a good catch! I actually purposely used `string` because you used to not be able to properly type check empty strings in this context, but it seems like the issue was fixed in a recent version of TypeScript. I'll try it out